install/kubernetes/helm/istio/charts/security/values.yaml

```diff
@@ -46,3 +46,11 @@ enableNamespacesByDefault: true
 # "security" and value "S1".
 podAntiAffinityLabelSelector: []
 podAntiAffinityTermLabelSelector: []
+
+# This parameter sets the CLI parameter append-dns-names of
+# Citadel, which:
+# - If true, Citadel will generate webhook certificates.
+# - If true, Citadel will not generate webhook certificates.
+# If there is another component generate webhook certificates,
+# this parameter should be set as false.
+appendDnsNames: true
```

Comment:
Why is the option named `appendDnsNames` when this option is about webhook config? I don't think it needs to directly line up with the CLI flag. This is user facing, while the cli flag isn't really

Reply:
Per the comment, the name of the option changed to enableGenerateWebhookCerts.